src/server/requestHandlers.js

```diff
@@ -117,7 +117,8 @@ sjrk.storyTelling.server.handleGetStory = function (request, dataSource) {
     var noAccessErrorMessage = "An error occurred while retrieving the requested story";
 
     promise.then(function (response) {
-        if (response.published) {
+        // only returns published stories or stories that predate the "published" flag
+        if (typeof response.published === "undefined" || response.published) {
```

Comment:
I do worry a bit about defaulting to published. If there are any issues with storing the story, it may get published by accident. Actually would this also publish those empty stories?

Reply:
I agree completely!

This solution was meant to get the stories back in action long enough to give us time to add that record to all the older stories, then revert back to the full restriction.

Reply:
I agree the proper solution is to migrate old records in the database with the new field of `"published": true` to avoid adding this kind of special handling which opens a potential hole in the code logic.

However, in case this temporary solution is required for the pressing release, it would be nice to add a TODO comment with the JIRA number to explain the plan. You may also wanna keep SJRK-425 open until the proper solution is implemented.